code/modules/uplink/uplink_items.dm

```diff
@@ -1487,9 +1487,10 @@ datum/uplink_item/stealthy_weapons/taeclowndo_shoes
 /datum/uplink_item/device_tools/singularity_beacon
 	name = "Power Beacon"
 	desc = "When screwed to wiring attached to an electric grid and activated, this large device pulls any \
-			active gravitational singularities or tesla balls towards it. This will not work when the engine is still \
-			in containment. Because of its size, it cannot be carried. Ordering this \
-			sends you a small beacon that will teleport the larger beacon to your location upon activation."
+			active gravitational singularities or tesla balls towards it (provided they are not safely \
+			contained), as well as increasing the odds of incoming meteor waves. \
```

Comment:
Meteors dont move tho. The station is moving ICly, thats why the immovable rod is immoveable

Reply:
In reality, there is no such thing as an immovable object or an unstoppable force anyway. Think of this as a disaster magnet and spare yourself from these peeves.